tasks/connectors/aqua/lib/aqua_helper.rb

```diff
@@ -5,10 +5,26 @@
 module Kenna
   module Toolkit
     module AquaHelper
-      SAAS_AUTH_URL = "https://api.cloudsploit.com/v2/signin"
-      WP_URL_API    = "https://prov.cloud.aquasec.com/v1/envs"
+      def select_region_urls(aqua_url)
+        case aqua_url
+        when /eu-1\.cloud\.aquasec\.com/
+          @saas_auth_url = "https://eu-1.api.cloudsploit.com/v2/signin"
+          @wp_url_api = "https://prov-eu-1.cloud.aquasec.com/v1/envs"
+        when /asia-1\.cloud\.aquasec\.com/
+          @saas_auth_url = "https://asia-1.api.cloudsploit.com/v2/signin"
+          @wp_url_api = "https://prov-asia-1.cloud.aquasec.com/v1/envs"
+        when /ap-2\.cloud\.aquasec\.com/
+          @saas_auth_url = "https://ap-2.api.cloudsploit.com/v2/signin"
+          @wp_url_api = "https://prov-ap-2.cloud.aquasec.com/v1/envs"
+        else
+          @saas_auth_url = "https://api.cloudsploit.com/v2/signin"
+          @wp_url_api = "https://prov.cloud.aquasec.com/v1/envs"
+        end
+      end
 
       def aqua_get_token(aqua_url, username, password)
+        select_region_urls(aqua_url)
+
         if cloud_url?(aqua_url)
           get_token_from_cloud(username, password)
```

Comment:
You only need to `select_region_urls` if it's a cloud connector, right? That could move inside this if.

Reply:
Or every call to a url accessor checks if it's cloud. I realize some of my suggestions are mutually exclusive. Just trying to draw your attention to some readability/maintainability issues.